lib/pure/math.nim

```diff
@@ -56,7 +56,7 @@ type
     fcNegInf     ## value is negative infinity
 
 proc classify*(x: float): FloatClass =
-  ## classifies a floating point value. Returns `x`'s class as specified by
+  ## Classifies a floating point value. Returns `x`'s class as specified by
```

Comment:
This is something that @Araq will disagree on. The standard library docs not beginning a sentence is a convention (to be fair, one which I don't follow, because I disagree with it). We need to have a proper discussion about this.